infra/feast-operator/internal/controller/services/util.go

```diff
@@ -6,13 +6,37 @@ import (
 
 	"github.com/feast-dev/feast/infra/feast-operator/api/feastversion"
 	feastdevv1alpha1 "github.com/feast-dev/feast/infra/feast-operator/api/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 )
 
 func isLocalRegistry(featureStore *feastdevv1alpha1.FeatureStore) bool {
 	appliedServices := featureStore.Status.Applied.Services
 	return appliedServices != nil && appliedServices.Registry != nil && appliedServices.Registry.Local != nil
 }
 
+func hasPvcConfig(featureStore *feastdevv1alpha1.FeatureStore, feastType FeastServiceType) (*feastdevv1alpha1.PvcConfig, bool) {
+	services := featureStore.Status.Applied.Services
+	var pvcConfig *feastdevv1alpha1.PvcConfig = nil
+	if feastType == OnlineFeastType && services.OnlineStore != nil && services.OnlineStore.Persistence.FilePersistence != nil {
```

Comment:
not an issue, how about considering `switch` statment here
   ```
 switch feastType {
    case OnlineFeastType:
        if services.OnlineStore != nil {  
```

Reply:
@tchughesiv ? I also thought of using the `switch` but I wanted to be consistent with the existing code

Reply:
i think that would be a better approach, so def go for it